rbac/management/principal/view.py

```diff
@@ -183,12 +182,13 @@ def get(self, request):
                 count = len(data)
             else:
                 count = None
+            last_link = int(count) - int(limit) if (int(count) - int(limit)) >= 0 else 0
```

Comment:
please consider better name for the variable, it should describe the value that is inside